docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+---
+title: NuGet 5.0 RTM Release Notes
+description: Release notes for NuGet 5.0 including known issues, bug fixes, new features, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 04/02/2019
+ms.topic: conceptual
+---
+
+# NuGet 5.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0](https://visualstudio.microsoft.com/downloads/) | 2.1.602<sup>1</sup>, 2.2.202<sup>2</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload 
+
+<sup>2</sup>Available as an optional install with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.0
+
+* Support solution filter when restoring the solution via CLI or in Visual Studio - [#5820](https://github.com/NuGet/Home/issues/5820)
+* `BuildTransitive` folder enables packages to transitively contribute targets/props to the host project - [#6091](https://github.com/NuGet/Home/issues/6091)
+* Better support for PackageReference scenarios in NuGet IVs APIs - [#7005](https://github.com/NuGet/Home/issues/7005), [#7493](https://github.com/NuGet/Home/issues/7493)
+* `dotnet pack project.json` has been deprecated - [#7928](https://github.com/NuGet/Home/issues/7928)
+* Gen 1 Credential Provider plugin is no longer supported - [#7819](https://github.com/NuGet/Home/issues/7819)
```

Comment:
It's not really no longer supported, but just depracated. We haven't removed it.

Reply:
@karann-msft  - Perhaps you just use the title of that issue, which is now:
Raise a deprecation warning for Gen1 credential plugin

Reply:
upodated it to say
> Gen 1 Credential Provider plugin has been superseded by [Gen 2](https://aka.ms/nuget-cross-platform-authentication-plugin) and will soon be deprecated - [#7819](https://github.com/NuGet/Home/issues/7819)